metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simp-simp",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
agreed, I'll bump it. This is a good example of why having a split develop/master can be a good thing, gets you an unstable integration branch that can churn without bumping vers, particularly on ideas that aren't fully proven out yet.